third_party/opa/data-and-policies/meshfordata-external-data/taxonomies.json

```diff
@@ -1,5 +1,5 @@
 {
-    "DataPurposes": ["audit&complience", "client Support", "marketing", "Fraud Detection", "Customer Behaviour Analysis"],
+    "DataIntents": ["audit&complience", "client Support", "marketing", "Fraud Detection", "Customer Behaviour Analysis"],
```

Comment:
It is actually data usage intent. Maybe rename to Intents and Roles?

Reply:
there is already DataRoles defined below. Wont it be confusing to name it DataIntentAndRoles? or did I understand your comment correctly ?

Reply:
I meant to rename: DataRoles -> Roles and DataIntents -> Intents

Reply:
Fixed. Please check. Thanks.